Makefile.work

```diff
@@ -582,6 +583,7 @@ SONIC_BUILD_INSTRUCTION :=  $(MAKE) \
                            SONIC_ENABLE_BOOTCHART=$(ENABLE_BOOTCHART) \
                            INCLUDE_FIPS=$(INCLUDE_FIPS) \
                            ENABLE_FIPS=$(ENABLE_FIPS) \
+                           ENABLE_DASH_SAI=$(ENABLE_DASH_SAI) \
```

Comment:
this introduce this kvm image as build option?

but can we include the docker-dash-engine in kvm vs image by default, then we can enable and disable it at runtime?

Reply:
I have no idea to always enable this for syncd-vs, Because as far as I know, we can only choose ONE SAI in a platform. It means if we decide to link the dash sai to syncd, the original vs-sai will be conflicted. It's also the reason why I introduced a build option. Because I have to choose a specific SAI in the build stage.
 
Yes, we can install the dash-engine by default, but I feel it doesn't make sense. Because the default kvm image doesn't include the dash sai, and I don't know how to make two SAI co-existing in one image.